sdk/python/feast/infra/feature_servers/aws_lambda/app.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import StrictBool, StrictStr
+from pydantic.typing import Literal
+
+from feast.repo_config import FeastConfigBaseModel
+
+
+class AwsLambdaFeatureServerConfig(FeastConfigBaseModel):
+    """Feature server config for AWS Lambda."""
+
+    type: Literal["aws_lambda"] = "aws_lambda"
+    """Feature server type selector."""
+
+    enabled: StrictBool
```

Comment:
Can we have a default value of False here?
```suggestion
    enabled: StrictBool = False
```